smtlib-backends-process/src/SMTLIB/Backends/Process.hs

```diff
@@ -107,6 +108,18 @@ new config = do
           )
     reportError' = (reportError config) . LBS.fromStrict
 
+-- | Send a command to the process without reading its response.
+write :: Handle -> Builder -> IO ()
+write handle cmd = do
+  hPutBuilder (getStdin $ process handle) $ cmd <> "\n"
+  IO.hFlush $ getStdin $ process handle
+
+-- | Send the @(exit)@ command to the process.
+-- Note that using @'write' ('toBackend' handle) "(exit)"@ instead could hang
+-- the process as it would wait for a response.
+exit :: Handle -> IO ()
+exit = flip write "(exit)"
```

Comment:
I think it would be clearer to ask the user to `Process.write "(exit)"` himself.

IIUC, calling `Process.exit` will leave a zombi process until the exit code is read with `wait`.